README.md

```diff
@@ -1,79 +1,69 @@
-# Hang In There
+# Hang In there
 
-A boilerplate repo. 
+### Developers
+- [Kristi Miller](https://github.com/Kristiannmiller)
+- [Estelle Staffieri](https://github.com/Estaffieri)
+### Project Manager
+- [Scott Ertmer](https://github.com/sertmer)
+### Project Links
+- [Repo](https://github.com/Kristiannmiller/hang-in-there-boilerplate)
+- [Deployed Page](https://kristiannmiller.github.io/hang-in-there-boilerplate/)
 
-## Set Up
+## Set-up
+- Fork the repo and clone down into terminal
+- Open in text editor
+- Read the README
+- In terminal, run command "open index.html" to interact with app.
 
-1. One teammate: fork this repository
-2. Go to settings and turn on GitHub Pages for this repository
-3. All teammates: clone down this repository
-4. `cd` into the repository
-5. Run `open index.html` to view it in the browser
+## Overview & Learning Goals
+For our mod 1 paired project, we will build a webpage that randomly generates motivational posters with quotes. It should, eventually, also allow user a user save their own posters as well.
 
-## Progression
-
-### Iteration 0 - Main Page
-
-![screenshot of main page showing poster](/readme-imgs/homepage.png)
-
-- When the page loads, we should see a poster with a randomly selected image, title, and quote
+We will develop our skills and understanding in JS functionality and changing webpage views.
 
-### Iteration 1 - Switching Views
+Our estimated schedule is as follows:
 
-Form page:
-![screenshot of form](/readme-imgs/form.png)
+- **7/07-08/2020**
+- **7/09-10/2020**
+- **7/11-12/2020**
+- **7/13-14/2020**
 
-Saved posters page (once working with extra saved posters):
-![screenshot of saved posters page](/readme-imgs/saved.png)
-
-- When a user clicks the "Make Your Own Poster" button, we should see the form, and the main poster should be hidden
-- When a user clicks the "View Saved Posters" button, we should see the saved posters area, and the main poster should be hidden
-- When a user clicks the "Nevermind, take me back!" or "Back to Main" buttons, we should only see the main poster section
-- In summary: Be able to switch between the three views (main poster, form, and saved posters) on the correct button clicks
-
-_Hint: go check out the HTML and CSS files to see how the form and saved posters sections are being hidden in the first place_
-
-## Iteration 2 - Creating a New Poster
+## Progression
```

Comment:
Are we going to fill this out as we go?

Reply:
Yes! We will update what we've already put the time in and start a log of what we accomplish every project workday!